job/runner.go

```diff
@@ -133,3 +186,44 @@ func (j *JobRunner) collectStats(success bool) {
 	j.currentStat.Success = success
 	j.currentStat.NumberOfRetries = j.job.Retries - j.currentRetries
 }
+
+func (j *JobRunner) checkExpected(statusCode int) bool {
+	// If no expected response codes passed, add 200 status code as expected
+	if len(j.job.RemoteProperties.ExpectedResponseCodes) == 0 {
+		j.job.RemoteProperties.ExpectedResponseCodes = append(j.job.RemoteProperties.ExpectedResponseCodes, 200)
+	}
+	for _, expected := range j.job.RemoteProperties.ExpectedResponseCodes {
+		if expected == statusCode {
+			return true
+		}
+	}
+
+	return false
+}
+
+// responseTimeout sets a default timeout if none specified
+func (j *JobRunner) responseTimeout() time.Duration {
+	responseTimeout := j.job.RemoteProperties.Timeout
```

Comment:
@levrado @ajvb Eventually making this a duration rather than an untyped number that represents seconds would be good